.gitignore

```diff
@@ -40,9 +40,9 @@ terraform.tfstate.d
 test/app/grpc_client
 test/loadtime/build
 test/e2e/build
+test/e2e/data/
 test/e2e/networks/*/
 test/logs
-test/p2p/data/
```

Comment:
I've noticed this path `test/p2p/data` referenced in a couple other places:

- in `test/docker/Dockerfile` 
```
RUN cometbft testnet \
  --config $REPO/test/docker/config-template.toml \
  --node-dir-prefix="mach" \
  --v=4 \
  --populate-persistent-peers=false \
  --o=$REPO/test/p2p/data
```

- in `.dockerignore` there's an entry for it there too

Reply:
I suspect the `test/docker` folder is no longer maintained and actually needs to be removed in its entirety.

Reply:
I failed to see those entries in `test/docker`. Thanks for pointing this out, @andynog! 

This `Dockerfile` is built on `golang:1.15` so it's rather old. It is used for the integration tests (`make test_integrations`) which I think they are unmaintained. 

So for the moment I suggest to leave `test/p2p/data/` in gitignore and just a new one for `test/e2e/data/`, which is actually needed. Removing `test/p2p/data/` should be part of another issue focusing on integration tests.

Reply:
Here's an existing issue on integration tests: #656